port/win32/omrsysinfo.c

```diff
@@ -323,6 +323,19 @@ WIN32_WINNT version constants :
 				PPG_si_osType = defaultTypeName;
 				isServerMajorVersion10 = TRUE;
 			} else
+#else /* defined(_WIN32_WINNT_WIN10) && (_WIN32_WINNT_MAXVER >= _WIN32_WINNT_WIN10) */
+			versionInfo.dwOSVersionInfoSize = sizeof(versionInfo);
+/* GetVersionEx() is deprecated, but still needed when using older compilers. Suppress the warning. */
+#pragma warning( suppress : 4996 )
+			if (GetVersionEx((OSVERSIONINFO *) &versionInfo)) {
```

Comment:
I think it's better to continue to detect the Windows version using the following code, and possibly report the wrong version, rather than fail to report anything.